### PR TITLE
Split test into more smaller tests

### PR DESCRIPTION
**Pull request by Koncpa**

### CODE REVIEW

File: Sanity/DAST_test/main.fmf [hunk 1]
```diff
@@ -0,0 +1,30 @@
+summary: DAST test for the tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/DAST_test/main.fmf:3]: Please, lets take this opportunity to change the contact

File: Sanity/configuration_test/main.fmf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+summary: Configuration test of tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/configuration_test/main.fmf:3]: Please, lets take this opportunity to change the contact

File: Sanity/functional_test/main.fmf [hunk 3]
```diff
@@ -0,0 +1,26 @@
+summary: Functionality test for the tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/functional_test/main.fmf:3]: Please, lets take this opportunity to change the contact

File: Sanity/key_management_test/main.fmf [hunk 4]
```diff
@@ -0,0 +1,26 @@
+summary: Key management test for the tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/key_management_test/main.fmf:3]: Please, lets take this opportunity to change the contact

File: Sanity/legacy_test/main.fmf [hunk 5]
```diff
@@ -0,0 +1,26 @@
+summary: Legacy test for the tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/legacy_test/main.fmf:3]: Please, lets take this opportunity to change the contact

File: Sanity/malware_detection_test/main.fmf [hunk 6]
```diff
@@ -0,0 +1,30 @@
+summary: Malware detection test for the tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/malware_detection_test/main.fmf:3]: Please, lets take this opportunity to change the contact

File: Sanity/scalability_test/main.fmf [hunk 7]
```diff
@@ -1,4 +1,4 @@
-summary: Basic functionality of the tang operator
+summary: Scalability test for the tang operator
 description: ''
 contact: Martin Zelený <mzeleny@redhat.com>
```
Comment (sarroutbi) [inline Sanity/scalability_test/main.fmf:3]: Please, lets take this opportunity to change the contact

**Review by sarroutbi — CHANGES_REQUESTED** [review]
Changes in general are great. Minor changes requested. Thanks Patrik

**Comment by sarroutbi** [comment]
@Koncpa : quick question about this change ... Do old instructions apply? If not ... I guess we have two options:
1 - Include a Makefile at Sanity directory that expands to execute all tests
2 - Change README.md to document which is the new procedure

I prefer the first one, but I am open to changes

**Comment by sopos** [comment]
It would be good to require the helper's directory explicitly, for the future compatibility once the https://github.com/teemtee/tmt/pull/2333 is merged and enabled by default.

```
require:
  - type: file
    pattern: /TestHelpers/.*
```
Ideally somewhere up in the structure so all the tests can inherit that. (Do not forget co use `require+`, in the tests.)

File: Sanity/DAST_test/main.fmf (hunk 1 above)
Comment (Koncpa) [reply Sanity/DAST_test/main.fmf]: Sure, DONE.

**Review by sarroutbi — APPROVED** [review]
Hello Patrik. Thanks for your PR. Changes LGTM

**Pull request merged**